lablqml/src_tests/test3.qml

```diff
@@ -0,0 +1,45 @@
+import QtQuick 2.1
+import QtQuick.Controls 1.0
+
+/* Demonstraction of direct property binding
+OCaml binds to objects named "test" and "mirror".
+When test changes, the binding function is called and the
+mirror's text value is set */
+
+ApplicationWindow {
+    visible: true
+    QtObject {
+	id: t
+	objectName: "test"
+	property string msg: "Hello world! Hello Qml with OCaml!"
+
+	// QtObjects can only be nested as properties
```

Comment:
Yes. But in QML not every thing needs to be nested using `property ..`. You can nest with declaring a nested Item.